config/samples/run_v1alpha3_kappcontrollerconfig.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: run.tanzu.vmware.com/v1alpha3
+kind: KappControllerConfig
+metadata:
+  name: kappcontrollerconfig-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
You might want to add sample config here?

Reply:
Added